client/package.json

```diff
@@ -19,7 +19,7 @@
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
     "//": "The sed command here is a temporary workaround to fix an issue in graphql-codegen",
-    "generate": "cd ../server && cargo run --bin remote_server_cli -- export-graphql-schema && cd ../client && graphql-codegen --config codegen.yml && find . -type f -name 'operations.generated.ts' -exec sed -i '' s/'graphql-request\\/dist\\/types\\.dom'/'graphql-request\\/src\\/types\\.dom'/g {} +",
+    "generate": "cd ../server && cargo run --bin remote_server_cli -- export-graphql-schema && cd ../client && graphql-codegen --config codegen.yml && find . -type f -name 'operations.generated.ts' -exec perl -i -pe 's/graphql-request\\/dist\\/types\\.dom/graphql-request\\/src\\/types\\.dom/g' {} \\;",
```

Comment:
I kind of missed this in the previous PR, I would have suggested something like this (with a comment, I think tsconfig.json allows comments):

```diff
diff --git a/client/tsconfig.json b/client/tsconfig.json
index a3acdeb8a..dd1110ab5 100644
--- a/client/tsconfig.json
+++ b/client/tsconfig.json
@@ -9,7 +9,10 @@
       "@common/intl/*": ["packages/common/src/intl/*"],
       "@common/styles": ["packages/common/src/styles"],
       "@common/types": ["packages/common/src/types"],
-      "@common/utils": ["packages/common/src/utils"]
+      "@common/utils": ["packages/common/src/utils"],
+      "graphql-request/dist/types.dom": [
+        "node_modules/graphql-request/src/types.dom"
+      ]
     },
     "resolveJsonModule": true,
     "target": "es2015",
```

Reply:
🤯  that is devious genius! am about to test it..

Reply:
snuck that in to 3467237a0
it's just too good not to